crates/model/Cargo.toml

```diff
@@ -9,22 +9,21 @@ license = "MIT OR Apache-2.0"
 doctest = false
 
 [dependencies]
-anyhow = "1"
-bigdecimal = "0.3"
-chrono = { version = "0.4", default-features = false, features = ["serde", "clock"] }
-derivative = "2.2"
-ethabi = "15.0"
```

Comment:
not actually needed since `ethabi` is re-exported through `web3` (and we already use it elsewhere).